server/models/db/student.js

```diff
@@ -10,7 +10,21 @@ const Tables = require('../../constants/tables.json');
 async function getStudent(email, password) {
     return userModel.getUser(email, password, Tables.student, Student);
 }
+// This method inserts a student account given specific information.
+async function insertStudentAccount(first_name, last_name, type, student_id, email, password, phone, school_id) {
+    //Data recieved from the front end sign up form
+    //Insert one row into a table
```

Comment:
Hi Nancy,
If possible, I would recommend right clicking and using the Format option in Visual Studio code to better align in the code for greater code readability.

Reply:
Done :)